src/main/java/stream/flarebot/flarebot/database/RedisController.java

```diff
@@ -27,7 +27,7 @@ public RedisController(JSONConfig config) {
                 config.getString("redis.host").get(),
                 Integer.parseInt(config.getString("redis.port").get()),
                 3000,
-                config.getString("redis.password").get());
+                config.getString("redis.password").get().isEmpty() ? null : config.getString("redis.password").get());
```

Comment:
Has this been tested? Jedis Doesn't like Nulls a lot of the time

Reply:
It does here since that basically means no password is set. Yes this has been tested due to me running it all the time for my test bot.

Reply:
Fair enough :+1: